src/components/DetailsPage/types.ts

```diff
@@ -15,4 +16,5 @@ export type DetailsPageTabProps = {
   partial?: boolean;
   className?: string;
   isFilled?: boolean;
+  unstableFeature?: FlagKey;
```

Comment:
Single tab can have multiple feature flags enabled
```suggestion
  featureFlags?: FlagKey[];
```